pcli/src/main.rs

```diff
@@ -32,7 +32,7 @@ const VIEW_FILE_NAME: &str = "pcli-view.sqlite";
 
 #[derive(Debug)]
 pub struct App {
-    pub view: ViewProtocolClient<BoxGrpcService>,
+    pub view: Option<ViewProtocolClient<BoxGrpcService>>,
```

Comment:
It'd be helpful to add a comment here explaining the meaning of the Option: that it's None for commands that can run offline, and Some for commands that can't.